e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-submit.spec.js

```diff
@@ -110,6 +117,18 @@ describe("Insurance - Your business - Company details page - As an Exporter I wa
         });
     });
 
+    it('should focus to the company website section when clicking the error', () => {
```

Comment:
I think this can be removed as it's also in the previous test?

Reply:
done